tests/ballerina-test-utils/src/main/java/org/ballerinalang/test/bala/BalaCreator.java

```diff
@@ -179,7 +179,7 @@ public static CompileResult compileWithTestsAndWrite(Path sourceRootPath, String
 
         BLangPackage bLangPackage = (BLangPackage) compileResult.getAST();
         Compiler compiler = Compiler.getInstance(context);
-        compiler.write(bLangPackage, targetPath);
+        //compiler.write(bLangPackage, targetPath);
```

Comment:
This class already deprecated but still, some functions used. We need to use use {@link org.ballerinalang.test.BCompileUtil} instead.